.eslintrc.json

```diff
@@ -1,134 +1,158 @@
+      "ignoreComments": false
+    }],
+    "no-param-reassign": ["error", {
+      "props": false
+    }],
+    "arrow-body-style": ["error", "as-needed"],
+    "arrow-parens": ["warn", "as-needed", {
+      "requireForBlockBody": false
+    }],
+    "arrow-spacing": ["error", {
+      "before": true,
+      "after": true
+    }],
+    "no-const-assign": "error",
+    "no-underscore-dangle": ["error", {
+      "allow": ["__REDUX_DEVTOOLS_EXTENSION_COMPOSE__"]
+    }],
+    "no-unused-expressions": [2, {
+      "allowShortCircuit": true
+    }],
+    "no-dupe-class-members": "error",
+    "no-duplicate-imports": "error",
+    "no-new-symbol": "error",
+    "no-this-before-super": "error",
+    "no-useless-computed-key": "error",
+    "no-useless-constructor": "warn",
+    "no-useless-rename": ["error", {
+      "ignoreDestructuring": false,
+      "ignoreImport": false,
+      "ignoreExport": false
+    }],
+    "no-var": "warn",
+    "object-curly-newline": ["error", {
+      "consistent": true
+    }],
+    "prefer-arrow-callback": ["warn", {
+      "allowUnboundThis": true
+    }],
+    "prefer-const": ["error", {
+      "destructuring": "all",
+      "ignoreReadBeforeAssign": true
+    }],
+    "prefer-template": "error",
+    "require-yield": "error",
+    "rest-spread-spacing": ["error", "never"],
+    "template-curly-spacing": "error",
+    "yield-star-spacing": ["error", "after"],
+    "prefer-spread": "error",
+    "generator-star-spacing": ["error", {
+      "before": false,
+      "after": true
+    }],
+    "symbol-description": "error",
+    "max-len": ["error", {
+      "code": 100
+    }],
+    "sort-keys": ["error", "asc"],
+    "react/display-name": [0, {
+      "ignoreTranspilerName": true
+    }],
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+    "react/no-danger": "error",
+    "react/jsx-pascal-case": "error",
+    "react/jsx-closing-bracket-location": "error",
+    "react/destructuring-assignment": [0, {
+      "ignoreClassFields": false
+    }],
+    "react/jsx-filename-extension": [1, {
+      "extensions": [".js", ".jsx"]
+    }],
+    "react/jsx-indent": ["error", 4],
+    "react/jsx-indent-props": ["error", 4],
+    "react/void-dom-elements-no-children": "error",
+    "react/jsx-tag-spacing": "error",
+    "react/jsx-no-literals": "error",
+    "react/jsx-wrap-multilines": "error",
+    "react/jsx-no-comment-textnodes": "error",
+    "react/prefer-stateless-function": [0, {
+      "ignorePureComponents": true
+    }],
+    "react/require-default-props": [0, {
+      "forbidDefaultForRequired": false
+    }],
+    "react/prop-types": [0, {
+      "ignore": true
+    }]
   },
   "settings": {
-      "react": {
-          "version": "latest"
-      }
+    "react": {
+      "version": "latest"
+    }
   }
-}
\ No newline at end of file
+}
```

Comment:
Add these changes in a separate PR, it has nothing to with your login code and bloats your PR

Reply:
I never intended formatting the code spacing. I did not on time that the files i visited were formatted on save. I only realised when i was about commiting my new change